03.bowling/bowling.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+require 'debug'
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a
+
+pins_of_tenth = frames.slice!(9..-1)
+frame_of_tenth = []
+pins_of_tenth.map do |pin|
+  pin = [10] if pin == [10, 0]
+  frame_of_tenth += pin
+end
+frames[9] = frame_of_tenth
+
+point = 0
+
+frames.each_with_index do |frame, idx|
+  point += frame.sum 
+  if idx <= 8 # 10フレーム目はスペア・ストライクボーナス無しのため除外
```

Comment:
```suggestion
  break if idx == 9 # 10フレーム目はスペア・ストライクボーナス無しのため除外
```

とした方がコメントとマッチしそうです。

あと、8よりも9を使った方が「配列は0始まりだから1少ないんだね」というのが推測しやすいと思います。

Reply:
breakでくり返しを止めれば良いんですね！
そちらの表現の方が、コメントアウトともやってることが一致してますね！